### ISSUE TITLE
burnt qkc when (minor block reward or tx fee) * tax_rate is not integer

### ISSUE THREAD

**Issue by qcdll** [QuarkChain/pyquarkchain#307]
this is not a serious bug, and normally won't be a problem, but...

assume tax rate is 0.5, and minor block reward or tx fee is for example 10001e-18 qkc:
minor block miner gets 5000e-18 qkc, and root block gets 5000e-18 qkc, meaning we lost 1e-18 qkc during the process

**Comment by qizhou** [comment]
Yes, that is possible, but if we lose 1e-18 QKC every ms, we will spend 1e15 / 3600 / 24 / 365 = 31709791.9838 years to burn one QKC.

**Comment by qcdll** [comment]
wontfix for now
